pkgs/top-level/all-packages.nix

```diff
@@ -18214,4 +18214,8 @@ with pkgs;
   rustdesk-flutter = callPackage ../by-name/ru/rustdesk-flutter/package.nix {
     flutter = flutter324;
   };
+
+  libvisual-full = callPackage ../by-name/li/libvisual/package.nix {
```

Comment:
Please try to sort this somewhere and don't append it at the end. If we would be doing that all the time we would have constant merge conflicts.